coralproject/talk
# Community Section - Search Commenters

# Community Section - Search Commenters

## What does this PR do?
- Adds the Community Section
- Search Commenters by Email and Display Name
- Change sorting by clicking Table Header
- Paginated at 50
- Ordered by creation date (we need to implement Signed up in the future)
- No Results state
- Loading States
- Pagination (it's working, you can request by page and send tons of parameters. But it's not final. I'll give it another refactor when we have more data (and time.)
- Redux
- Adds version Number

## How do I test this PR?
- Do `npm install` just in case
- `npm start` && `npm run build`
- Go to `http://localhost:3000`
- Go to Community (`http://localhost:3000/admin/community`)
- You should be able to see the community section and search by pressing enter and do all of the above

Note: I did some styling to match the designs but we are going to address design and pixel perfect in another PR in the future.

@coralproject/tech
